src/lib/utils/exceptions.ts

```diff
@@ -0,0 +1,8 @@
+export function handleFetchException(
```

Comment:
Should maybe be more explicit with the function name here, e.g.
```suggestion
export function logException(
```

Reply:
I wanted to explicitly use this helper for fetch exceptions so what about `logFetchException`?

The current implementation can be valid for any other kind of exception but that could change if we decide to give more fetch related info in the future. Maybe a bit YAGNI but not too much effort.